pypose/sparse/sparse_block_tensor.py

```diff
@@ -134,20 +141,27 @@ def repeated_value_as_hybrid_value(coo, block_shape, val_func):
 
     # Number of block and number of elements per block.
     n_block = coo.values().shape[0]
-    n_block_elem = block_shape[0] * block_shape[1]
 
     # Prepare the sequence number.
-    val = val_func(n_blk=n_block, device=coo.device)
-    val = val.repeat_interleave(n_block_elem).view((n_block, *block_shape))
+    if mode == 'zeros':
+        val = zeros(n_block, device=coo.device, dtype=dtype)
+    elif mode == 'ones':
+        val = ones(n_block, device=coo.device, dtype=dtype)
+    elif mode == 'sequence':
+        val = arange(n_block, device=coo.device, dtype=dtype)
+    else:
+        raise ValueError(f'Unknown mode: {mode}')
+    val = val[..., None, None].tile(block_shape)
 
     # Only use the sparse dimension.
-    s_dim = coo.shape[:2]
+    s_dim: List[int] = list(coo.shape[:2])
 
-    return torch.sparse_coo_tensor(
-            coo.indices(), val, size=(*s_dim, *block_shape)
+    return sparse_coo_tensor(
+            coo.indices(), val, size=s_dim + block_shape
```

Comment:
Is `list(coo.shape[:2])` required by JIT. Also, is `s_dim + block_shape` required by JIT?